Dockerfile

```diff
@@ -27,4 +27,8 @@ RUN chmod 755 /usr/local/bin/runserver.sh
 COPY ./ $HOME/go-api/
 WORKDIR $HOME/go-api/
 
+RUN \
+      cd /usr/local/lib/python*/site-packages/django/contrib/admin/ ; \
+      patch < /home/ifrc/go-api/main/sites_patch
+
```

Comment:
Hey @szabozoltan69 I don't think this is a good approach. Please see this discussion. The first solution mentioned is what I would recommend:

https://stackoverflow.com/questions/40781374/how-change-django-admin-view-site-link-to-custom-absolute-url

ie in `api/admin.py` add a line like so:

`admin.site.site_url = "https://" + os.environ.get('FRONTEND_URL')`